File: infrastructureservice/PlatformCI.InfrastructureService.Models/Implementation/IisSite.cs
using System;
using System.Collections.Generic;

namespace PlatformCI.InfrastructureService.Models.Implementation
{
    [Serializable]
    public class IisSite
    {
        public string SiteName { get; set; }
        public string PhysicalPath { get; set; }
        public IisApplicationPool AppPool { get; set; }
        public Guid SiteGuid { get; set; }
        public IList<string> Environments { get; set; }
    }
}

File: models/inframodel/iissite.go
package inframodel

type IisSite struct {
	SiteName     string
	PhysicalPath string
	AppPool      IisApplicationPool
	SiteGuid     string
	Applications []IisApplication
	Environments []EnvironmentPart
}

type IisSitePart struct {
	SiteName     string
	PhysicalPath string
	SiteGuid     string
	Environments []EnvironmentPart
}
